docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -422,3 +422,98 @@ Use the SQL Server Authentication option and keep the Password field empty:
 
 Click OK to connect.
 
+## Cloud Spanner DataGrip
+
+The Cloud Spanner integration works only in the local proxy tunnel mode.
+Start a local proxy tunnel for connections to your Spanner database by using the
+command below:
+
+```
+tsh proxy db --tunnel --port 1443 spanner --db-user=<service-account-name> --db-name=<spanner database name>
+```
+
+The `--db-user` should be the name, minus the
+"@project-id.iam.gserviceaccount.com" suffix, of a GCP service account that can
+access the Spanner database.
```

Comment:
This sentence reads awkwardly IMO. We start describing the name and then side track the reader mentioning the suffix before we tell them it's the name of a service account.